src/openrct2/interface/Viewport.h

```diff
@@ -111,8 +111,8 @@ void viewports_invalidate(int32_t left, int32_t top, int32_t right, int32_t bott
 void viewport_update_position(rct_window* window);
 void viewport_update_sprite_follow(rct_window* window);
 void viewport_update_smart_sprite_follow(rct_window* window);
-viewport_focus viewport_update_smart_guest_follow(rct_window* window, Peep* peep);
-void viewport_update_smart_staff_follow(rct_window* window, Peep* peep);
+viewport_focus viewport_update_smart_guest_follow(rct_window* window, const Peep* peep);
+void viewport_update_smart_staff_follow(rct_window* window, const Peep* peep);
```

Comment:
```suggestion
void viewport_update_smart_staff_follow(rct_window* window, const Staff* staff);
```
Makes a bit more sense I think